cmd/watch.go

```diff
@@ -52,12 +52,12 @@ var watchCmd = &cobra.Command{
 			defer watcher.Stop()
 			signalChan := make(chan os.Signal)
 			signal.Notify(signalChan, os.Interrupt)
-			return watch(ctx, watcher.Events, signalChan)
+			return watch(ctx, watcher.Events, signalChan, checksums)
```

Comment:
This is the map of paths to checksums as fetched from the server upon running `watch`.

Reply:
Why pass in a map? We could pass in the previous checksum here from the event right?